packages/kbn-monaco/src/esql/lib/ast/autocomplete/autocomplete.ts

```diff
@@ -444,9 +441,6 @@ function isFunctionArgComplete(
   }
   const hasCorrectTypes = fnDefinition.signatures.some((def) => {
     return arg.args.every((a, index) => {
-      if (def.infiniteParams) {
-        return true;
-      }
```

Comment:
I think it is okay to remove this. All three functions that were using `infiniteParams` already declare their types as `any`